docs/tutorials/projects/guardian.md

```diff
@@ -0,0 +1,850 @@
+
+At this point also connect the speaker to your Raspberry Pi.
+
+Then test the components on the [robot's Control tab](/manage/fleet/robots/#control) again to ensure everything still works.
+
+## Detect persons and pets
+
+For the guardian to be able to detect living beings, you can use [this Machine Learning model](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/effdet0.tflite).
+The model can detect a variety of things which you can see in the associated <file>[labels.txt](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/labels.txt)</file> file.
+
+You can also [train your own custom model](/manage/ml/train-model/) based on images from your robot but the provided Machine Learning model is a good one to start with.
+
+To use the provided Machine Learning model, copy the <file>[effdet0.tflite](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/effdet0.tflite)</file> file and the <file>[labels.txt](https://github.com/viam-labs/devrel-demos/raw/main/Light%20up%20bot/labels.txt)</file> to your Raspberry Pi:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+scp effdet0.tflite pi@guardian.local:/home/pi/effdet0.tflite
+scp labels.txt pi@guardian.local:/home/pi/labels.txt
+```
+
+{{< tabs >}}
+{{% tab name="Builder UI" %}}
+
+Next, navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Services** subtab and navigate to the **Create service** menu.
+
+1. **Add a ML service.**
+
+   The [ML model service](/services/ml) allows you to deploy the provided machine learning model to your robot.
+   Create an ML model with the name `mlmodel`, the type `mlmodel` and the model `tflite_cpu`.
+   Then click **Create Service**.
+
+   In the new ML Model panel, select **Path to Existing Model On Robot** for the **Deployment**.
+
+   Then specify the absolute **Model Path** as `/home/pi/effdet0.tflite` and the **Label Path** as `/home/pi/labels.txt`.
+
+2. **Add a vision service.**
+
+   Next, add a [detector](/services/vision/detection) as a vision service to be able to make use of the ML model.
+   Create an vision service with the name `detector`, the type `vision` and the model `mlmodel`.
+   Then click **Create Service**.
+
+   In the new detector panel, select the `mlmodel` you configured in the previous step.
+
+   Click **Save config** in the bottom left corner of the screen.
+
+3. **Add a `transform` camera.**
+
+   To be able to test that the vision service is working, add a `transform` camera which will add bounding boxes and labels around the objects the service detects.
+
+   Click on the **Components** subtab and navigate to the **Create component** menu.
+   Create a [transform camera](/components/camera/transform) with the name `transform_cam`, the type `camera` and the model `transform`.
+
+   Replace the attributes JSON object with the following:
```

Comment:
If you have time here a bit more explanation of what's happening with the pipeline attributes and transform config could be helpful-- I was initially confused as to why attributes were being replaced with attributes, explaining that it's a "transformation object" could be useful!